### DISCUSSION TITLE
feat: ✨ syn-select: dropdown items should be same size as the select itself

### DISCUSSION THREAD

**Discussion by klaarseSICKAG** [synergy-design-system/synergy-design-system#526] [RFCs]
# Component

I suggest to add the feature to the following component: syn-select

# Description
If setting the syn-select size to small the dropdown is still medium. This looks really strange, especially as the font-size of the items are now larger than even titles.

![Image](https://github.com/synergy-design-system/synergy-design-system/assets/44837861/6a61da3c-cd93-436e-a01d-08a5862b3eca)

# Acceptance criteria

- [ ] All DoD items successfully completed
- [ ] Dropdown items have same size as the component itself

**Comment by klaarseSICKAG** [comment]
refers to: https://deagxjira.sickcn.net/jira/browse/SYN-13

**Comment by schilchSICKAG** [comment]
Fixed some time ago.
